<?xml version="1.0"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-1-g15f4949  Build Version: 1.6.7594.29634
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot name="morph_bot">
  <link name="base_link">
    <inertial>
      <origin
        xyz="-2.4606E-06 0.011027 -0.0037489"
        rpy="0 0 0" />
      <mass value="1.239" />
      <inertia
        ixx="0.022223"
        ixy="2.0374E-08"
        ixz="-9.7846E-09"
        iyy="0.024872"
        iyz="0.00013466"
        izz="0.020437" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh filename="package://morph_bot_description/urdf/mesh/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh filename="package://morph_bot_description/urdf/mesh/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint name="right_motor" type="continuous">
    <origin
      xyz="-0.031204 0 -0.019431"
      rpy="0.019516 0 3.1416" />
    <parent link="base_link" />
    <child link="right_wheel" />
    <axis xyz="0 0 -1" />
  </joint>
  <link name="left_wheel">
    <inertial>
      <origin
        xyz="0.0031767 6.6028E-08 9.7201E-08"
        rpy="0 0 0" />
      <mass value="0.20688" />
      <inertia
        ixx="7.8842E-05"
        ixy="4.1106E-11"
        ixz="1.3096E-11"
        iyy="4.0023E-05"
        iyz="4.5149E-10"
        izz="4.0023E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh filename="package://morph_bot_description/urdf/mesh/left_wheel.STL" />
      </geometry>
      <material name="">
        <color rgba="0.094118 0.094118 0.094118 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh filename="package://morph_bot_description/urdf/mesh/left_wheel.STL" />
      </geometry>
    </collision>
  </link>
  <joint name="left_motor" type="continuous">
    <origin
      xyz="0.031204 0 -0.019431"
      rpy="1.2608 0 0" />
    <parent link="base_link" />
    <child link="left_wheel" />
    <axis xyz="0 0 1" />
  </joint>
  <link name="right_wheel">
    <inertial>
      <origin
        xyz="0.0031767 6.6225E-08 9.744E-08"
        rpy="0 0 0" />
      <mass value="0.20688" />
      <inertia
        ixx="7.8842E-05"
        ixy="4.0951E-11"
        ixz="1.3204E-11"
        iyy="4.0023E-05"
        iyz="4.515E-10"
        izz="4.0023E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh filename="package://morph_bot_description/urdf/mesh/right_wheel.STL" />
      </geometry>
      <material name="">
        <color rgba="0.094118 0.094118 0.094118 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh filename="package://morph_bot_description/urdf/mesh/right_wheel.STL" />
      </geometry>
    </collision>
  </link>
  <joint name="right_motor" type="continuous">
    <origin
      xyz="-0.031204 0 -0.019431"
      rpy="0.019516 0 3.1416" />
    <parent link="base_link" />
    <child link="right_wheel" />
    <axis xyz="0 0 -1" />
  </joint>
</robot>